### ISSUE TITLE
sendmail issue

### ISSUE THREAD

**Issue by hamahamahama** [guileen/node-sendmail#72]
Hello @GreenPioneer ,
I used the sendmail API in order to send a mail with node js , so I followed your documentation  as we can see in the picture , as a result no error appears and i can't receive any email . 

![image](https://user-images.githubusercontent.com/23224377/70041582-bb15ed00-15bd-11ea-91bb-27730aa99e34.png)

Can you Help me please ?

**Comment by GreenPioneer** [comment]
@hamahamahama 

Have you tried sending from a server not just your localhost?

Google often blocks emails coming from personal computers

**Comment by himharsh1997** [comment]
i don't think so as I had used nodemailer for report sending and it send to gmail without get blocked once.

**Comment by skywa04885** [comment]
Port 25, 587 or 465 blocked by ISP maybe ? On some ISP's it just refuses to connect, others just look like they connect, but just hang... Just like trying to connect to a protected port on a UFW firewall.
